t/t3431-rebase-ignore-ws.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+#
+# Copyright (c) 2019 Rohit Ashiwal
+#
+
+test_description='git rebase --ignore-whitespace tests'
+
+. ./test-lib.sh
+
+cat >patch1.patch <<EOF
+diff --git a/main.c b/main.c
+new file mode 100644
+--- /dev/null
++++ b/main.c
+@@ -0,0 +1,6 @@
++#include <stdio.h>
++
++int main() {
++	printf("Hello World\n");
++	return 0;
++}
+EOF
+
+sed -e 's/Z/ /g' >patch2.patch <<EOF
+diff --git a/main.c b/main.c
+--- a/main.c
++++ b/main.c
+@@ -3,4 +3,4 @@
+Zint main() {
+-	printf("Hello World\n");
++	printf("Hello Git\n");
+Z	return 0;
+Z}
+EOF
+
+sed -e 's/	/        /g' <patch1.patch >patch3.patch
```

Comment:
All these setup steps should still be in an `test_expect_success` block.  While we're not testing Git here, any potential output gets swallowed up only inside of `test_expect_success` (or `test_expect_failure` for that matter), so when running tests without verbose mode, should any of the lines above print any output, I think we might end up breaking the TAP format, which allows running the tests under `prove`.

Even if we think the above can never actually print any output it's still better to err on the side of caution and keep everything inside the `test_expect_success` blocks.